### PR TITLE
Docs fixes based on Mopinion feedback

### PR DESCRIPTION
**Pull request by bethaitman**
- Removing old introduction
- Adding instructions for how to update your PATH (the links seemed reasonable to me, but shout if they don't seem right to you)
- Update quickstart screenshots and contract numbers, which seem to have changed

### CODE REVIEW

File: docs/source/getting-started/quickstart.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ You can get the quickstart application using the DAML assistant (``daml``):
 
 #. Run ``daml new quickstart quickstart-java``
 
-   This creates the ``quickstart-java`` application into a new folder called ``quickstart``.
+   This creates the ``quickstart-java`` application i nto a new folder called ``quickstart``.
```
Comment (cocreature) [inline docs/source/getting-started/quickstart.rst]: That looks like a weird change?

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -27,6 +27,8 @@ To install the SDK on Mac or Linux:
      curl -sSL https://get.daml.com/ | sh
 2. If prompted, add ``~/.daml/bin`` to your PATH.
 
+   If you don't know how to do this, try following `these instructions for MacOS <https://hathaway.cc/2008/06/how-to-edit-your-path-environment-variables-on-mac/>`_ or `these instructions for Windows <https://www.java.com/en/download/help/path.xml>`_.
```
Comment (cocreature) [inline docs/source/getting-started/installation.rst]: I wonder if we should add this to the prompt message instead of in the docs (or possibly both)?

Comment (bethaitman) [reply docs/source/getting-started/installation.rst]: I think it'd be a nice addition to the prompt message. As long as the links are stable that seems reasonable - @fmota-da , what do you think?

File: docs/source/getting-started/quickstart.rst (hunk 1 above)
Comment (bethaitman) [reply docs/source/getting-started/quickstart.rst]: weird, that was an accident

**Pull request merged**